Instructor-Guide--Assignments--Marking--Grading-View.md

```diff
@@ -43,9 +44,50 @@ The grading bar, located at the top of the grading view page, provides some usef
 
 ![Grading Bar](images/grade-view-grading-bar.png)
 
-The left half of the bar allows you to quickly switch between assignments by clicking on the "Previous" and "Next" buttons. It also displays the group number of the group who's submission you're currently grading. Finally, on the right, it shows you how many submissions have been set to the "Complete" status.
+The left half of the bar allows you to quickly switch between submissions by clicking on the "Previous" and "Next" buttons. Filters for these submissions can be specified by graders by clicking the "Filter Submissions" button (see "[Filter Submissions](#filter-submissions)"). A grader can also view a random incomplete submission by clicking the "Random Incomplete Submission" button. Further, the grading bar displays the group number of the group who's submission you're currently grading. Finally, on the right, it shows you how many submissions have been set to the "Complete" status.
```

Comment:
Instead of "a random incomplete submission" say "a random submission whose grading is incomplete". (The wording of "Random Incomplete Submission" is a bit ambiguous but it's okay for the UI, and we can make it clearer here.)

Also, in a subsequent sentence I noticed there's a "who's" that should read "whose"